aspnet/identity/overview/features-api/customize_identity_model.md

```diff
@@ -0,0 +1,807 @@
+---
+title: Identity model customization
+author: ajcvickers
+description: This article describes how to customize the underlying Entity Framework Core data model for ASP.NET Core Identity.
+manager: elipton
```

Comment:
@Rick-Anderson `manager`, `ms.prod`, `ms.technology`, and `ms.topic` should be removed. These pull from the *docfx.json* file now.

Reply:
Left manager since it's not the default. But we should probably remove that too.